etl/pypasar/db/sql/postgres/provider/provider.sql

```diff
@@ -0,0 +1,45 @@
+-- *******************************************************************
+-- NAME: provider.sql
+-- DESC: Final table - provider
+-- *******************************************************************
+-- CHANGE LOG:
+-- DATE        VERS  INITIAL  CHANGE DESCRIPTION
+-- ----------  ----  -------  ----------------------------------------
+-- 2024-09-07  1.00           Initial create
+-- 2024-09-07  2.00           Insert data into provider table from staging view
+-- 2024-09-07 3.00            Updated the schema name
+-- *******************************************************************
+
+
+INSERT INTO {OMOP_SCHEMA}.provider
+(
+    provider_id,
+    provider_name,
+    npi,
+    dea,
+    specialty_concept_id,
```

Comment:
For the `specialty_concept_id` field, please refer to the specification to map to the corresponding ID: https://sidata.plus/PASAR_ETL_Spec/provider.html

Reply:
Pending concept id mapping

Reply:
Oh, I see. Based on the meeting discussion, we could also consider joining with `source_to_concept_map` for the specialty_concept_id field.